landing-page/content/services/row-level-deletes.html

```diff
@@ -1,9 +1,11 @@
 ---
-Title: Schema Evolution
-AsciinemaCast: schema_evolution.cast
+Title: Row-level Deletes
+Description: Iceberg supports the copy-on-write and merge-on-read approaches to handle row-level delete operations.
+LottieFile: row-level-deletes-animation.json
```

Comment:
For this one as well, I'm not sure how well the animation represents row-level deletes. I would much rather highlight the MERGE INTO syntax in a SQL box. That's a really useful thing to see!

Reply:
Renamed this to "Expressive SQL" and used the code example below:
```
MERGE INTO prod.nyc.taxis pt
USING (SELECT * FROM staging.nyc.taxis) st
ON pt.id = st.id
WHEN NOT MATCHED THEN INSERT *;
```

relevant commit: [b92aebb](https://github.com/apache/iceberg-docs/pull/18/commits/b92aebb3e4550189467840c9c052e13295afc727)